MainWindow_New.py

```
@@ -1,742 +1,742 @@
+		self.canvas = canvas#父控件
+		self.value = text
+		self.tag = text
+ 
+		self.x1 = pos[0] #左上角x坐标
+		self.y1 = pos[1] #左上角y坐标
+		self.x2 = pos[2] #右下角x坐标
+		self.y2 = pos[3] #右下角y坐标
+		self.d_outline = d_outline#默认外框颜色
+		self.d_fill = d_fill#默认文字颜色
+ 
+		self.rec = self.canvas.create_rectangle(x1,y1,x2,y2,width=2,outline=self.d_outline,tag=self.tag)
```

Ruff (0.14.8)

`x1` may be undefined, or defined from star imports

(F405)